imnodes.cpp

```diff
@@ -2044,111 +2051,126 @@ void EditorContextMoveToNode(const int node_id)
 
 void Initialize()
```

Comment:
`Initialize()` would become an internal function, and would be used to set the state of the given pointer, instead of the global.

```suggestion
void Initialize(Context* ctx)
```